doc/getting_started/installation_mac.rst

```diff
@@ -9,6 +9,7 @@ After completing these steps, you will be able to compile and run your Zephyr
 applications on the following Mac OS version:
 
 * Mac OS X 10.11 (El Capitan)
+* macOS Sierra 10.12
 
 Developing for Zephyr on OS X generally requires you to build the
```

Comment:
Looks like the Mac OS is now called "macOS" so the rest of these references to "OS X" should probably be change to "macOS", yes?